### PR TITLE
Genres

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
Cerated an Emoji Component.  Background colour is hardcoded and will need to be refactored once tailwind config is done.
Enums were used

### CODE REVIEW

**Comment by NoHara42** [comment]
Which Issue does this relate to?

**Comment by NoHara42** [comment]
Tailwind setup is on main, please reverse merge

**Comment by TheDoplarEffect** [comment]
Have merged main on my machine and added tailwind classes to component

File: src/stories/Emogi.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Emogi } from "../components/Emogi";
+import { Meta, StoryFn } from "@storybook/react";
+import "../index.css";
```
Comment (NoHara42) [inline src/stories/Emogi.stories.tsx:4]: you shouldn't need to import this here

**Review by NoHara42 — CHANGES_REQUESTED** [review]
remove -lock.json files and see small change

**Pull request closed without merging**